src/main/java/com/autotune/operator/KruizeDeploymentInfo.java

```diff
@@ -38,20 +37,36 @@
  */
 public class KruizeDeploymentInfo {
     private static final Logger LOGGER = LoggerFactory.getLogger(KruizeDeploymentInfo.class);
-    private static String clusterType;
-    private static String kubernetesType;
-    private static String authType;
-    private static String authToken;
-    private static String monitoringAgent;
-    private static String monitoringAgentService;
-    private static String monitoringAgentEndpoint;
-    private static String loggingLevel;
-    private static String rootLoggingLevel;
+    public static String DATABASE_ADMIN_USERNAME;
+    public static String DATABASE_ADMIN_PASSWORD;
+    public static String DATABASE_USERNAME;
+    public static String DATABASE_PASSWORD;
+    public static String DATABASE_HOSTNAME;
+    public static String DATABASE_DBNAME;
+    public static String DATABASE_PORT;
+    public static String DATABASE_SSL_MODE;
+    public static String SETTINGS_SAVE_TO_DB;
+    public static String SETTINGS_DB_DRIVER;
+    public static String SETTINGS_HIBERNATE_DIALECT;
+    public static String SETTINGS_HIBERNATE_CONNECTION_DRIVER_CLASS;
+    public static String SETTINGS_HIBERNATE_C3P0_MIN_SIZE;
+    public static String SETTINGS_HIBERNATE_C3P0_MAX_SIZE;
+    public static String SETTINGS_HIBERNATE_C3P0_TIMEOUT;
+    public static String SETTINGS_HIBERNATE_C3P0_MAX_STATEMENTS;
+    public static String SETTINGS_HIBERNATE_HBM2DDL_AUTO;
+    public static String SETTINGS_HIBERNATE_SHOW_SQL;
+    public static String AUTOTUNE_MODE;
+    public static String MONITORING_AGENT;
+    public static String MONITORING_SERVICE;
+    public static String MONITORING_AGENT_ENDPOINT;
+    public static String CLUSTER_TYPE;
+    public static String K8S_TYPE;
+    public static String AUTH_TYPE;
+    public static String AUTH_TOKEN;
```

Comment:
Do these vars now have some special significance? It looks like they are also being over loaded as ENV params?

Reply:
yes this values can be set either via config json or via ENV.